### PR TITLE
*: move 'Project detail' to op-guide

### PR DESCRIPTION
**Pull request by gyuho**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,37 +127,3 @@ See [CONTRIBUTING](CONTRIBUTING.md) for details on submitting patches and the co
...
-
-To avoid inadvertently running a possibly unstable etcd server, `etcd` on unsupported architectures will print
-a warning message and immediately exit if the environment variable `ETCD_UNSUPPORTED_ARCH` is not set to
-the target architecture.
-
-Currently only the amd64 architecture is officially supported by `etcd`.
-
-[358]: https://github.com/coreos/etcd/issues/358
-
-### License
```
Comment (xiang90) [inline README.md]: we should keep the license in the readme.

File: Documentation/docs.md [hunk 2]
```diff
@@ -28,6 +28,7 @@ Administrators who need to create reliable and scalable key-value stores for the
  - [Understand failures][failures]
  - [Disaster recovery][recovery]
  - [Performance][performance]
+ - [Project detail][project]
```
Comment (xiang90) [inline Documentation/docs.md]: probably we should have two sections:

- [versioning]
- [supported platform]

Comment (gyuho) [inline Documentation/docs.md]: Ok I will separate them out.

**Comment by gyuho** [comment]
@xiang90 PTAL. All fixed. Thanks.

**Comment by xiang90** [comment]
lgtm

**Comment by gyuho** [comment]
thanks.

**Pull request merged**